app/app.php

```diff
@@ -1,6 +1,63 @@
 <?php
 
+function checkMaxLength($number, $value) {
+	if (is_numeric($number) ) {
+		if(strlen($value) >= $number) {
+			return false;
+		}else {
+			return true;
+		}
+	}else {
+		die("validate check max length rule invalid");
+	}
+	
+}
+function checkMinLength($number, $value) {
+	if (is_numeric($number) ) {
+		if(strlen($value) >= $number) {
+			return true;
+		}else {
+			return false;
+		}
+	}else {
+		die("validate check min length rule invalid");
+	}
+}
```

Comment:
Dùng if else tối ưu hơn đi e, dùng toán tử cũng đc 
```suggestion
function checkMinLength($number, $value) {
       return is_numeric($number) ? strlen($value) >= $number : die("validate check min length rule invalid");
}
```

Reply:
ok a.